packages/@unimodules/react-native-adapter/ios/UMReactNativeAdapter/Services/UMReactNativeAdapter.m

```diff
@@ -130,7 +132,11 @@ - (void)unregisterAppLifecycleListener:(id<UMAppLifecycleListener>)listener
 
 - (JSGlobalContextRef)javaScriptContextRef
 {
-  return _bridge.jsContextRef;
+  if ([_bridge respondsToSelector:@selector(jsContextRef)]) {
+    return _bridge.jsContextRef;
+  } else {
+    return *(((JSGlobalContextRef*)(_bridge.runtime )) + 1);
```

Comment:
Would you, please, add some explanatory comment here why do we have to do such magic?